### PR TITLE
Allow setting `prefixText` dynamically

### PR DESCRIPTION
**Pull request by yorkie**
This adds the possibility to prepend a timestamp on the log.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ Text to display after the spinner.
 
 ##### prefixText
 
-Type: `string`
+Type: `string | () => string`
 
-Text to display before the spinner. No prefix text will be displayed if set to an empty string.
+Text or text generator to display before the spinner. No prefix text will be displayed if set to an empty string.
```
Comment (sindresorhus) [inline readme.md:51]: Not clear enough what a "text generator" is.

The docs changes here should also be reflected in index.d.ts

Comment (yorkie) [reply readme.md:51]: How about "Text or function to return a string to display before the spinner"?

Comment (sindresorhus) [reply readme.md:51]: Text or a function that returns text

Comment (yorkie) [reply readme.md]: I have updated the readme and `index.d.ts` :)

**Pull request merged**